docs/AboutUs.md

```diff
@@ -4,49 +4,37 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [OH HAN GYEOL](https://github.com/Leook0209)
+<img src="images/han.jpg" width="150"><br>
+Role: Developer <br>
```

Comment:
you need to indicate who is in charge of which component as well. e.g., storage, logic etc.,